terraform/aks/variables.tf

```diff
@@ -30,6 +42,18 @@ variable "postgres_flexible_server_sku" { default = "B_Standard_B1ms" }
 variable "postgres_enable_high_availability" { default = false }
 variable "azure_enable_backup_storage" { default = true }
 
+variable "replicas" { default = 1 }
+variable "memory_max" { default = "1Gi" }
+
 locals {
   azure_credentials = try(jsondecode(var.azure_credentials), null)
+  kv_app_secrets    = yamldecode(data.azurerm_key_vault_secret.app_secrets.value)
+  app_resource_group_name = "${var.azure_resource_prefix}-${var.service_short}-${var.config_short}-rg"
+  app_secrets = merge(
+    local.kv_app_secrets,
+    {
+      DATABASE_URL          = module.postgres.url
+      REDIS_CACHE_URL       = module.redis-cache.url
```

Comment:
Are these necessary if the app is using the connection strings instead?

Reply:
Was just following the same standard as register app....will remove